Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -2070,9 +2069,8 @@ public IDataConsolidator ResolveConsolidator(Symbol symbol, TimeSpan? timeSpan)
             // data we won't be able to do anything good, we'll call it second, but it would really just be minute!
             if (timeSpan.Value < subscription.Resolution.ToTimeSpan())
             {
-                throw new ArgumentException(string.Format("Unable to create {0} consolidator because {0} is registered for {1} data. " +
-                                                          "Consolidators require higher resolution data to produce lower resolution data.",
-                    symbol, subscription.Resolution)
+                throw new ArgumentException($"Unable to create {symbol} consolidator because {symbol} is registered for " +
+                    $"{subscription.Resolution} data. Consolidators require higher resolution data to produce lower resolution data."
```

Comment:
Resolution is missing the .ToStringInvariant?

Reply:
I've reviewed this entire file and updated all relevant cases to use the `using static` pattern. A nice approach (methinks) was creating an overload for `CreateIndicatorName` that accepts a `FormattableString` (the interpolated string) and then performs the `Invariant(fs)` to minimize required edits and the noise that would have been added if we did `Invariant(fs)` in each invocation to `CreateIndicatorName`